scripts/install.sh

```diff
@@ -534,47 +534,12 @@ function ask_to_continue() {
 
 }
 
-# Get changelog for specific version
-# Only version description and breaking changes are taken
+# Get changelog for specific branch
 function get_changelog() {
```

Comment:
Maybe `show_changelog_link` now instead?

Reply:
changed to `print_changelog_link`